### DISCUSSION TITLE
How to install mod

### DISCUSSION THREAD

**Discussion by Tyler8x** [DaXcess/LCVR#3] [Q&A] [answered]
How can I install this mod to give it a go myself? I know it isn't finished, but I'd still like to try the current build if that's possible

**Comment by DaXcess** [comment, marked as answer]
Using the mod at this stage, even when just trying it out is very not recommended as **the stereoscopic rendering of the camera is a facade.**
![image](https://github.com/DaXcess/lethalcompany-vr-plugin/assets/46288749/45a6561b-6377-4be2-9320-d9a6d1a9945d)
![image](https://github.com/DaXcess/lethalcompany-vr-plugin/assets/46288749/bbaa440a-eea1-41ac-877a-30e30339c025)

In the above image I talked about "a subtle drift" but that was before I noticed that perspective is completely destroyed with the current setup. A new camera system is in the works, including motion controls however this code is not yet available (and unplayable too at this stage).

**Reply by Tyler8x** [reply]
ah thank you, I'll keep watching for when it's a bit more stable. Cheers 👍
